Sources/SwiftIDEUtils/SyntaxClassification.swift

```diff
@@ -128,6 +128,8 @@ extension RawTokenKind {
       return .none
     case .floatingLiteral:
       return .floatingLiteral
+    case .hashbang:
+      return .poundDirective
```

Comment:
I’m not sure that makes sense because it’s not a pound directive in the Swift sense. I would just classify it as `none`.

Reply:
Oh, that's because I found hashbang is colored the same as other macros in Xcode so I think we could classify it as `.poundDirective` and make it highlighted the same as other macros in sourcekit-lsp.

Reply:
Hmm, I think we can deviate here. `poundDirective` might make sense now but it makes less sense if we rename `poundDirective` to `ifConfigDirective` as discussed offline. 

If you do want to color it, I would color it as a line comment, because that’s kind-of what it is: The `#` indicates a comment in bash, I think that’s what the `#!` has evolved from and Swift just supports it as well.

Reply:
Ah, that makes sense. I'll just leave it as `none` for now.